libs/openFrameworks/utils/ofFileUtils.cpp

```diff
@@ -110,7 +110,8 @@ void ofBuffer::clear(){
 //--------------------------------------------------
 void ofBuffer::allocate(long _size){
 	clear();
-	buffer.resize(_size);
+	//we always add a 0 at the end to avoid problems with strings
+	buffer.resize(_size + 1);
```

Comment:
actually can you change it to:

```
buffer.resize(_size + 1,0)
```

so the buffer is set to 0, else the final char can be anything and won't work as null terminator for strings